src/header.rs

```diff
@@ -279,25 +280,45 @@ impl NiftiHeader {
             .ok_or_else(|| NiftiError::InvalidCode("sform", self.sform_code as i16))
     }
 
+    /// Ensure that the current `descrip` field is valid and is exactly equal to 80 bytes.
```

Comment:
It might be worth documenting that "descriptions shorter than 80 bytes will be extended with trailing zeros".